client/src/elm/Pages/NCD/Activity/Utils.elm

```diff
@@ -57,10 +57,13 @@ expectActivity currentDate assembled db activity =
         MedicalHistory ->
             True
 
-        -- @todo
-        _ ->
+        Laboratory ->
             True
 
+        NextSteps ->
+            -- @todo
```

Comment:
Do we have the ability to sort it out now?

Reply:
@AronNovak NCD Next steps related development is done at #456 and #459, All commented out code is handled there.